supply/lib/supply/options.rb

```diff
@@ -210,6 +210,11 @@ def self.available_options
                                      description: "Whether to skip uploading SCREENSHOTS",
                                      type: Boolean,
                                      default_value: false),
+        FastlaneCore::ConfigItem.new(key: :sync_image_upload,
```

Comment:
The reason I didn't name it `sync_screenshots` is deliberate: this is because this not only applies to screenshots, but also to Play Store artwork — typically the `featuredGraphic` and `icon` images that are not screenshots but rather the banner and large icon shown in the Google Play's download page for the app.

I thought calling it `sync_screenshots` would make people think that this only applied to screenshots (as in `skip_upload_screenshots: false`) while it also applies to `skip_upload_images: false` too.
Arguably, because the app artwork is referenced by the term `…images…` in the `skip_upload_images` option, one could argue that calling my new parameter `sync_image_upload` would make people think that it applies to that option only (i.e. where `image` term here would be interpreted as "the artwork images uploaded as part of `skip_upload_images: false`), while the intent of the term `…image…` in `sync_image_upload` here is more to say "sync the upload of every kind of [PNG, JPEG, …] image file".

But I couldn't find a term that would encompass both "each image file, be it a screenshot or an artwork image" other than just the term "image", hence settling for this.

---

As for trying to make the `deliver` and `supply` option names in sync, that ship as sadly already sailed: For example, [`deliver` uses `skip_screenshots`](https://github.com/fastlane/fastlane/blob/master/deliver/lib/deliver/options.rb#L137) while [`supply` uses `skip_upload_screenshots`](https://github.com/fastlane/fastlane/blob/master/supply/lib/supply/options.rb#L207) (and [`skip_upload_images`](https://github.com/fastlane/fastlane/blob/master/supply/lib/supply/options.rb#L207)), with `_upload_` in that option name for `supply` while `deliver` doesn't have it 🤷

Reply:
Good points! 😅 I think you named it well 😊 Thanks for clarifying!